src/settings/tabconnection.h

```diff
@@ -0,0 +1,47 @@
+#ifndef TABCONNECTION_H
+#define TABCONNECTION_H
+
+#include <QWidget>
+#include <QtWidgets>
+
```

Comment:
Header QtWidgets includes all elements of Qt UI
It is huge
You must remove this include from your files, especially - from headers
**_It might cause identifiers conflict_**

Reply:
removed